homeassistant/components/mealie/strings.json

```
@@ -8,7 +8,7 @@
           "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]"
         },
         "data_description": {
-          "host": "The URL of your Mealie instance."
+          "host": "The URL of your Mealie instance, e.g. http://192.168.1.123:1234"
```

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg

```suggestion
          "host": "The URL of your Mealie instance, for example, http://192.168.1.123:1234"
```